src/main/java/Deadlines.java

```diff
@@ -0,0 +1,19 @@
+public class Deadlines extends Task{
```

Comment:
Is there a reason why the class name is in plural form?
This is evident in some other classes as well.